packages/react-native/scripts/cocoapods/jsengine.rb

```diff
@@ -35,7 +35,7 @@ def depend_on_js_engine(s)
   if ENV["USE_HERMES"] == nil || ENV["USE_HERMES"] == "1"
     s.dependency 'hermes-engine'
     s.dependency 'React-hermes'
-  else
+  elsif ENV['USE_THIRD_PARTY_JSC'] != '1'
     s.dependency 'React-jsc'
```

Comment:
I think we should leave it until we figure out how to pass `jsEngine` dependency name dynamically. The extracted Pod name is called `React-jsc` too

Reply:
good catch! i was removing the entire `s.dependency 'React-jsc'` when testing.